cmd/registry-replacer/main.go

```diff
@@ -93,6 +98,10 @@ func main() {
 	if err != nil {
 		logrus.WithError(err).Fatal("failed to gather options")
 	}
+	if opts.maxConcurrency == 0 {
+		opts.maxConcurrency = runtime.GOMAXPROCS(0)
```

Comment:
Can we use a fixed higher number like 500, this tool needs to make over 6k http requests currently?

Reply:
6000 / 4 would be 1.5k seconds, assuming each request takes one second and we never have to retry